static/js/modules/localStorage/localStorageSprint.js

```diff
@@ -0,0 +1,59 @@
+import {startRun, submitRun, updateAnonymousRun} from "../game/runs.js"
+import { getLocalStorageRuns, addRunToLocalStorage, setLocalStorageRuns } from "./localStorage.js"
+
+function startLocalRun(promptId, runId) {
```

Comment:
So this is stored by runId? What do you think about storing by promptId first?

Reply:
I had it based on runId since its guaranteed to be unique, and it makes more sense as an iterable for when we pop all entries and make an API call for each run. Would there be a benefit for storing by promptId?

Also there may be more than one run per prompt